src/dashboard/home-dashboard.component.tsx

```diff
@@ -27,7 +31,14 @@ export default function HomeDashboard(props: HomeDashboardProps) {
               </Link>
             </div>
             {config.buttons.enabled && (
-              <div className={styles.buttonArea}>{buttons}</div>
+              <div className={styles.buttonArea}>
+                <ExtensionSlotReact extensionSlotName="home-page-buttons">
```

Comment:
given that the name is plural "home-page-button**s**", do you expect multiple extensions to attach here? If there are multiple extensions, what determines the ordering?

Reply:
Yes I do, an arbitrary number of extensions can be attached to an extension slot. The order is determined by the order in which the extensions are passed in the call to `attach`.